File: check_today.py
import json
import mysql.connector

try:
    db = mysql.connector.connect(host='localhost',user='root',password='user',database='bhspl_attendance')
    c = db.cursor(dictionary=True)
    c.execute("SELECT * FROM raw_logs WHERE emp_id='15241' AND DATE(punch_time) = CURDATE() ORDER BY punch_time DESC")
    print("Sridhar Today:", json.dumps(c.fetchall(), default=str, indent=2))
    
    # Check for any names containing Yash
    c.execute("SELECT emp_id, emp_name FROM employees WHERE emp_name LIKE '%Yash%'")
    print("Yashwanth Search:", json.dumps(c.fetchall(), indent=2))
except Exception as e:
    print(e)

File: src/main/java/com/bhspl/tools/CheckToday.java
package com.bhspl.tools;

import java.sql.Connection;
import java.sql.PreparedStatement;
import java.sql.ResultSet;

public class CheckToday {
    public static void main(String[] args) {
        String q1 = "SELECT * FROM raw_logs WHERE emp_id='15241' AND DATE(punch_time) = CURDATE() ORDER BY punch_time DESC";
        String q2 = "SELECT emp_id, emp_name FROM employees WHERE emp_name LIKE '%Yash%'";
        try (Connection c = DbHelper.getConnection()) {
            try (PreparedStatement ps = c.prepareStatement(q1); ResultSet rs = ps.executeQuery()) {
                System.out.println("Sridhar Today:");
                DbHelper.printResultSet(rs);
            }
            try (PreparedStatement ps2 = c.prepareStatement(q2); ResultSet rs2 = ps2.executeQuery()) {
                System.out.println("Yashwanth Search:");
                DbHelper.printResultSet(rs2);
            }
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}
